test_cases/dispense_if_else.bs

```diff
@@ -0,0 +1,11 @@
+manifest fluid_a
+
+instructions:
+
+if (3<=3) {
+	a = dispense 5 units of fluid_a
+} else {
+	a = dispense 10 units of fluid_a
+}
+
+b = split a into 5
```

Comment:
All test cases belong in the `\tests\test_cases\volume\` folder.  You will then need to add a test harness to `\tests\frontend\` that handles volume tracking.  You can copy whatever of the boilerplate you need from other files.

Reply:
How do I access the global variables stored in my Pass's class? I don't see an obvious way of grabbing it using the existing functions in front_end_base.py